cli/internal/state/state.go

```diff
@@ -56,7 +56,7 @@ func NewFromIDFile(idFile clusterid.File, cfg *config.Config) *State {
 			UID:               idFile.UID,
 			ClusterEndpoint:   idFile.IP,
 			APIServerCertSANs: idFile.APIServerCertSANs,
-			InitSecret:        string(idFile.InitSecret),
```

Comment:
just curious: what was the reason to make it a byte slice?

Reply:
Initially I thought a string fits best as this essentially is just a random string that we receive from the applied Terraform config. However, at almost all places we use this, it needs to be a byte slice. And as it contains only printable characters either way, I figured to just save it as a byte slice directly to save on unnecessary conversions.